scout/parse/variant/coordinates.py

```diff
@@ -170,13 +170,20 @@ def parse_coordinates(variant, category, build="37"):
             end_chrom=end_chrom,
             svlen=variant.INFO.get("SVLEN"),
         )
+    elif category == "mei":
+        sub_category = "mei"
+        end = int(variant.end)
+        length = alt_len
+        if ref_len != alt_len:
+            sub_category = "mei"
+            length = abs(ref_len - alt_len)
     else:
         sub_category = "snv"
         end = int(variant.end)
         length = alt_len
         if ref_len != alt_len:
             sub_category = "indel"
-            abs(ref_len - alt_len)
+            length = abs(ref_len - alt_len)
```

Comment:
This looks like an old omission. Fortunately it does not seem to be used elsewhere?

Reply:
I'm not sure, but good catch!